contracts/zap/UniswapV2Zapper.sol

```diff
@@ -72,10 +72,10 @@ contract UniswapV2Zapper is IZapper, Swapper {
         uint balanceTokenA = IERC20Upgradeable(zapData.tokenA).balanceOf(address(this));
         uint balanceTokenB = IERC20Upgradeable(zapData.tokenB).balanceOf(address(this));
 
-        if (balanceTokenA > 0)
+        if (balanceTokenA > 99)
```

Comment:
why `99`? maybe leave a comment for better context or use a constant with a descriptive name